packages/migrate/src/__tests__/DbExecute.test.ts

```diff
@@ -24,7 +24,7 @@ describe('db execute', () => {
   describe('using Prisma Config', () => {
     it('--url is not supported', async () => {
       ctx.fixture('prisma-config-validation/sqlite-d1')
-      const config = (await loadConfigFromFile({ configFile: 'prisma.config.ts', configRoot: ctx.fs.cwd() })).config!
```

Comment:
In Jest, loadConfigFromFile cannot be used unless the experimental-vm-modules flag is enabled. However, enabling experimental-vm-modules requires the entire codebase to be compatible with ESM in order to run tests. Therefore, we handle this by loading the config directly instead.